jkube-kit/jkube-kit-wildfly-jar/src/main/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricher.java

```diff
@@ -55,8 +55,10 @@ private enum Config implements Configs.Config {
         SUCCESSTHRESHOLD("successThreshold","1"),
         LIVENESSINITIALDELAY("livenessInitialDelay", "60"),
         READINESSINITIALDELAY("readinessInitialDelay", "10"),
+        PERIODSECONDS("periodSeconds", "10"),
```

Comment:
Do you think we should also add a startupInitialDelay field?
```
STARTUPINITIALDELAY("startupInitialDelay", "10"),
```

Reply:
I just refactor periodSeconds to startupInitialDelay

Reply:
According to [Configure Probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes), the default value for `periodSeconds` is 10 and if we keep the default value then it's same as `startupInitialDelay` & `readinessInitialDelay` so, do we need to change any of these or keep the default?

Reply:
I think it's okay to keep the default values as provided by kubernetes. If the user needs to change these, an explicit configuration can be provided.